<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>紧固管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icons/TS.png</normaloff>:/icons/TS.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>工单列表</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>隐藏已完成</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>全部工单</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>数据操作</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>扳手数据</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>新建</string>
    </property>
    <addaction name="actNewJob"/>
    <addaction name="actNewTemp"/>
    <addaction name="actArrange"/>
   </widget>
   <widget class="QMenu" name="menuDB">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="actNewDB"/>
    <addaction name="actOpenDB"/>
    <addaction name="actOutputExcel"/>
    <addaction name="actPrint"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="actByJob"/>
    <addaction name="acByTool"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actSetWorker"/>
    <addaction name="actSetTool"/>
    <addaction name="actSetCoed"/>
    <addaction name="actLanguage"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuDB"/>
   <addaction name="menuQuery"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>紧固管理系统</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNewJob"/>
   <addaction name="actArrange"/>
   <addaction name="actNewTemp"/>
   <addaction name="separator"/>
   <addaction name="actDBs"/>
   <addaction name="actQuery"/>
   <addaction name="actSettings"/>
   <addaction name="separator"/>
   <addaction name="actAbout"/>
   <addaction name="acExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actNewJob">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/newJob.jpg</normaloff>:/icons/newJob.jpg</iconset>
   </property>
   <property name="text">
    <string>新建工单</string>
   </property>
  </action>
  <action name="actNewTemp">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/template.jpg</normaloff>:/icons/template.jpg</iconset>
   </property>
   <property name="text">
    <string>新建模板</string>
   </property>
  </action>
  <action name="actArrange">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/arrangement.JPG</normaloff>:/icons/arrangement.JPG</iconset>
   </property>
   <property name="text">
    <string>工单安排</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/about.jpg</normaloff>:/icons/about.jpg</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actDBs">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/DBfile.jpg</normaloff>:/icons/DBfile.jpg</iconset>
   </property>
   <property name="text">
    <string>数据</string>
   </property>
  </action>
  <action name="actQuery">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/query.jpg</normaloff>:/icons/query.jpg</iconset>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/settings.jpg</normaloff>:/icons/settings.jpg</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actNewDB">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>新建数据库</string>
   </property>
  </action>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actOutputExcel">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>导出Excel</string>
   </property>
  </action>
  <action name="actPrint">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actByJob">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>按工单</string>
   </property>
  </action>
  <action name="acByTool">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>按工具</string>
   </property>
  </action>
  <action name="acExit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/exit.jpg</normaloff>:/icons/exit.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actSetWorker">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>人员</string>
   </property>
  </action>
  <action name="actSetTool">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>工具</string>
   </property>
  </action>
  <action name="actSetCoed">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>代码</string>
   </property>
  </action>
  <action name="actLanguage">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/common.png</normaloff>:/icons/common.png</iconset>
   </property>
   <property name="text">
    <string>语言</string>
   </property>
   <property name="toolTip">
    <string>语言转换</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
